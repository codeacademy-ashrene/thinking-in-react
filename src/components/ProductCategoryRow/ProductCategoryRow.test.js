import React from 'react';
import ProductCategoryRow from '.';
import renderer from 'react-test-renderer';

describe('ProductCategoryRow', () => {
  it('should render the category of products', () => {
    const tree = renderer.create(<ProductCategoryRow category={"Sporting Goods"}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});