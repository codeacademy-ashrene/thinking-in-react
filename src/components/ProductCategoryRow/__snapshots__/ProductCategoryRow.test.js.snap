// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCategoryRow should render the category of products 1`] = `
<tr>
  <th>
    Sporting Goods
  </th>
</tr>
`;
